frontend/src/components/module-editor/LessonViewer.tsx

```diff
@@ -20,7 +20,12 @@ const LessonViewer = (): React.ReactElement => {
         {(provided) => (
           <Flex ref={provided.innerRef} direction="column" flex="1">
             {lesson?.content.map((block, index) => (
-              <EditableContentBlock block={block} key={index} index={index} />
+              <EditableContentBlock
+                block={block}
+                key={index}
+                index={index}
+                context={context}
```

Comment:
Instead of passing `context` all the way down, it would be better to add a `useContext` hook call inside `EditTextModal`.